README.md

```diff
@@ -154,19 +126,32 @@ Note: this is assuming that you clone this repo on your raspberry pi in the main
 
 ## Structure
 
-_           | filename                    | description
----------:  | :-----------                | :---------------------------------------------------
-**device**  | `pvm.py`                    | main python script, this runs on each pi device
-**control** | `max-init.txt`              | this file can make control patch setup faster
-_           | `pvm.maxpat`                | main control patch. controls 6 pvm devices at the same time
-_           | `pvm_control.maxpat`        | abstraction with the control patch GUI to be embedded as a bpatcher
-_           | `pvm_init.maxpat`           | abstraction responsible for parsing the `max-init.txt` file
-_           | `pvm_send.maxpat`           | abstraction for OSC sending. Arguments: _ip port_. Attributes: `@ip` `@port` 
-_           | `pvm_warmup.maxpat`         | abstraction for interpolating playback rates, to be embedded as a bpatcher
-_           | `pvm.maxproj`               | Max project file. Openning this file will load all main control patches.
-_           | `host.py`                   | (to be removed) control a remote device using a python script instead
-**others**  | `launch.sh`                 | shell script to start `pvm.py` with one _click_
-_           | `build_omxplayer.sh`                 | shell script to build `omxplayer` with one _click_
```

Comment:
ow... you just removed the table :( ... wasn't it so nice to look at?

Reply:
Add table back